pkg/storage/localfs/store.go

```diff
@@ -76,6 +76,7 @@ func (rc readCloser) Close() error {
 }
 
 func (l *localFS) Put(ctx context.Context, key string, source io.Reader, exclusive bool) error {
+	// TODO: Change this implementation to use rename to put file into place.
```

Comment:
see #162 for a decorator pattern approach

Reply:
Can you file an issue under harmonization? Don't want to miss this.